src/main/webapp/app/course/plagiarism-cases/shared/review/plagiarism-case-review.component.html

```diff
@@ -1,7 +1,14 @@
 <ul ngbNav #nav="ngbNav" class="nav-tabs row" [destroyOnHide]="false">
     <ng-container *ngFor="let submission of plagiarismCase.plagiarismSubmissions; let i = index">
+        <div class="mt-2">
+            {{ 'artemisApp.plagiarism.plagiarismCases.overallSimilarity' | artemisTranslate }}: <strong>{{ submission.plagiarismComparison.similarity?.toFixed(1) }}%</strong>
+        </div>
         <li [ngbNavItem]="i" class="col px-0">
             <a ngbNavLink class="btn-sm text-dark text-center">{{ 'artemisApp.plagiarism.plagiarismCases.comparison' | artemisTranslate }} {{ i + 1 }}</a>
+            <div class="my-2" style="font-size: large; text-align: center">
```

Comment:
Please do not use inline styles; either use bootstrap classes or define custom class an style the element in a ``scss`` file.

Reply:
Thanks, I addressed the issues (duplicate similarity, dark text, inline styles).